slack_bolt/listener/asyncio_runner.py

```diff
@@ -174,12 +174,11 @@ def _start_lazy_function(self, lazy_func: Callable[..., Awaitable[None]], reques
         copied_request = self._build_lazy_request(request, func_name)
         self.lazy_listener_runner.start(function=lazy_func, request=copied_request)
 
-    @staticmethod
-    def _build_lazy_request(request: AsyncBoltRequest, lazy_func_name: str) -> AsyncBoltRequest:
-        copied_request = create_copy(request.to_copyable())
-        copied_request.method = "NONE"
```

Comment:
mypy detected this property is not defined in the request class and actually the property is never used